packages/common/icons/src/lib/history-tab-icon/history-tab-icon.tsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable-next-line */
+import { SvgIcon, SvgIconProps } from '@mui/material';
+
+/* eslint-disable-next-line */
+export interface HistoryTabIconProps extends SvgIconProps {}
+
+export function HistoryTabIcon(props: HistoryTabIconProps) {
+  return (
```

Comment:
You have to camel case some of these props in react:
![image](https://github.com/user-attachments/assets/b92168f2-b27b-44ce-bf09-d863c23fa053)